.github/workflows/build-and-test.yml

```diff
@@ -0,0 +1,605 @@
+      - name: Build ${{ matrix.package_type }} arm64 package
+        run: ./internal/buildscripts/packaging/fpm/${{ matrix.package_type }}/build.sh "${{ steps.github_tag.outputs.tag }}" "arm64" "./dist/"
+      - name: Test ${{ matrix.package_type }} package
+        run: |
+          if [[ "${{ matrix.package_type }}" = "deb" ]]; then
+              ./internal/buildscripts/packaging/fpm/test.sh dist/otel-contrib-collector*amd64.deb examples/tracing/otel-collector-config.yml
+          else
+              ./internal/buildscripts/packaging/fpm/test.sh dist/otel-contrib-collector*x86_64.rpm examples/tracing/otel-collector-config.yml
+          fi
+      - name: Upload Packages
+        uses: actions/upload-artifact@v2
+        with:
+          name: collector-packages
+          path: ./dist/*
+  windows-msi:
+    runs-on: windows-latest
+    needs: [cross-compile]
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Download Binaries
+        uses: actions/download-artifact@v2
+        with:
+          name: collector-binaries
+          path: ./bin/
+      - name: Cache Wix
+        id: wix-cache
+        uses: actions/cache@v2
+        env:
+          cache-name: cache-wix
+        with:
+          path: |
+            "C:\Program Files (x86)\WiX Toolset v3.11"
+          key: wix-3.11
+      - name: Install Wix Toolset
+        if: steps.wix-cache.outputs.cache-hit != 'true'
+        run: .\internal\buildscripts\packaging\msi\make.ps1 Install-Tools
+      - run: mkdir -p dist
+      - name: Build MSI
+        run: |
+          $Version = if ($env:GITHUB_REF -match '^refs/tags/(\d+\.\d+\.\d+)') { $Matches[1] } else { "0.0.1" }
+          .\internal\buildscripts\packaging\msi\make.ps1 New-MSI -Version $Version
+      - name: Validate MSI
+        run: .\internal\buildscripts\packaging\msi\make.ps1 Confirm-MSI
+      - name: Upload MSI
+        uses: actions/upload-artifact@v2
+        with:
+          name: collector-packages
+          path: ./dist/*.msi
+
+  publish-check:
+    runs-on: ubuntu-latest
+    needs: [build-package, windows-msi]
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v2
+      - name: Download Binaries
+        uses: actions/download-artifact@v2
+        with:
+          name: collector-binaries
+      - name: Download Packages
+        uses: actions/download-artifact@v2
+        with:
+          name: collector-packages
+          path: ./dist/
+      - name: Verify Distribution Files Exist
+        id: check
+        run: ./.github/workflows/scripts/verify-dist-files-exist.sh
+  publish-dev:
+    runs-on: ubuntu-latest
+    needs: [lint, unittest, integration-tests, build-package, windows-msi]
+    if: ${{ false }}
+    # if: (github.ref == 'refs/heads/main' || startsWith(github.ref, 'refs/tags/')) && github.repository == 'open-telemetry/opentelemetry-collector-contrib'
```

Comment:
Same here: Can we delete commented code or turn it into a TODO comment?

Reply:
Done